invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login_user.html

```diff
@@ -27,21 +27,25 @@ <h3 class="ui login header">{{ _('Log in to Repository') }}</h3>
           <div class="ui divider"></div>
           <!--Log in with SSO-->
           {% if config.OAUTHCLIENT_REMOTE_APPS or config.SSO_SAML_IDPS or config.EDUGAIN_LOGIN_ENABLED %}
```

Comment:
do we need this, if we have a guard around every option?

Reply:
this a horizontal divider below that still needs this check
(it divides these logins from local logins and shows iff both - one of these logins, and local login - are enabled.

this would be more obvious if the conditional was closer to that divider that still uses it
Moved it down